src/sentry/profiles/flamegraph.py

```diff
@@ -239,3 +239,38 @@ def get_profile_ids_for_span_op(
     spans = [tup[1] for tup in transaction_to_prof.values()]
 
     return {"profile_ids": profile_ids, "spans": spans}
+
+
+def get_profiles_with_function(
+    organization_id: int,
+    project_id: int,
+    function_fingerprint: str,
+    params: ParamsType,
+):
+    query = Query(
+        match=Entity(EntityKey.Functions.value),
+        select=[
+            Function("groupUniqArrayMerge", [Column("examples")], "profile_ids"),
```

Comment:
I think this query will grow in memory without bounds. So we should change it to something like `groupUniqArrayMerge(100)` to limit the number of profile ids it returns.

Reply:
yep, thanks @Zylphrex !

I was indeed looking for that; an equivalent of the `LIMIT` clause for the  `groupUniqArrayMerge ` to limit the data at DB level instead of doing `data[0]["profile_ids"][:100]`